### PR TITLE
UI changed

### PR DESCRIPTION
**Pull request by ishika1501**
#1  issue resolved , the UI has been updated , sir please mention if any future changes are required.

### CODE REVIEW

File: public/css/style.css [hunk 1]
```diff
@@ -136,12 +140,12 @@ a {
...
 }
 .info {
-  font-family: var(--secondary-font);
+  font-family: 'Raleway', sans-serif;
   font-size: 30px;
   margin: 20px auto;
 }
 .success {
-  --primary-color: rgba(0, 143, 0, 0.767);
+  --primary-color: #DEB841;
```
Comment (pulkit-30) [inline public/css/style.css:148]: Ui doesn't respond!!!

**Review by pulkit-30 — CHANGES_REQUESTED** [review]

**Comment by ishika1501** [comment]
@pulkit-30 i have understood the things you mentioned , i will improve it and push it again , thankyou for your corporation. i am getting it done as soon as possible .Thankyou sir

**Review by Ayush7614 — CHANGES_REQUESTED** [review]
@ishika1501 Doing Changes Correctly

File: public/css/style.css [hunk 2]
```diff
@@ -1,22 +1,35 @@
 @import url("https://fonts.googleapis.com/css2?family=Mochiy+Pop+P+One&family=Rubik:wght@300&display=swap");
+<<<<<<< HEAD
```
Comment (pulkit-30) [inline public/css/style.css:2]: By mistake, you have pushed the conflicting code

**Review by pulkit-30 — CHANGES_REQUESTED** [review]

**Comment by pulkit-30** [comment]
Any Update?

File: public/css/style.css (hunk 2 above)
Comment (ishika1501) [reply public/css/style.css]: i am getting this done sir .sorry for that.

**Comment by ishika1501** [comment]
@pulkit-30  sir  hopefully i have resolved the issue , please accept it .

File: public/css/style.css [hunk 3]
```diff
@@ -136,13 +141,199 @@ a {
...
+    color: #fff;
+    font-size: 20px;
+    border-radius: 2px;
+    -webkit-border-radius: 5px;
+    -moz-border-radius: 5px;
+    -ms-border-radius: 5px;
+    -o-border-radius: 5px;
+    cursor: pointer;
+  }
+}
```
Comment (pulkit-30) [inline public/css/style.css:339]: Use Css variables for background colors
ref - https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

**Review by pulkit-30 — CHANGES_REQUESTED** [review]

**Comment by pulkit-30** [comment]
Great Work @ishika1501 🎉 
Thanks

**Pull request merged**